tests/unit_tests/pyanaconda_tests/modules/localization/test_module_localization.py

```diff
@@ -1088,3 +1233,137 @@ def test_localed_wrapper_no_systembus(self, mocked_system_bus):
         mocked_system_bus.check_connection.return_value = False
         localed_wrapper = LocaledWrapper()
         self._guarded_localed_wrapper_calls_check(localed_wrapper)
+
+
+class LiveSystemKeyboardTestCase(unittest.TestCase):
```

Comment:
Consider doing this in a new test file matching the new module file.

This whole file could use some splitting anyway, but that's another independent PR...

Reply:
If I do that I should also split localed test from this to keep consistency. However, not sure if it should be part of this PR (it is already big). Let's handle it as followup.

Reply:
Yes - #5029.